typescript/src/schema/proto.ts

```diff
@@ -44,6 +44,7 @@ interface MethodDescriptorProto
     | undefined;
   pagingFieldName: string | undefined;
   pagingResponseType?: string;
+  pagingMapResponseType?: string | undefined;
```

Comment:
No need to have `| undefined` since the question mark does just that.

Reply:
👍